### PR TITLE
Add section about downloading human genome

### PR DESCRIPTION
**Pull request by Lauren640**
This section just explains the wget command and displays a link for uploading the human genome

### CODE REVIEW

File: README_human.md [hunk 1]
```diff
@@ -11,3 +11,15 @@ This will usually mean opening a terminal and using `ssh` to connect. The genera
...
 	ssh username@my.hpc.server.edu.au
 ```
+
+## Downloading the human genome
+This step will be important when using the HISAT program to align the RNAseq reads onto the genome. This step will use the `wget` command, which allows the user to download particular files from the web. The general format of this command is:
+
+	```bash
+		wget http://website.come/files
+	```
+To download the *human genome*, enter the following command into the terminal on HPC
```
Comment (iracooke) [inline README_human.md]: It would be slightly more correct to say 

To download a version of the *human genome* with HiSat2 indexes, enter ...

**Comment by iracooke** [comment]
@Lauren640 Great.  Have you tested the wget command as shown?  Can you make the small change as per my comment please.

**Comment by Lauren640** [comment]
@iracooke Changes have been made. 
The wget command appears to be working, but still has another 26 minutes of downloading before I can say for sure.

**Comment by iracooke** [comment]
@Lauren640 Sounds good.  When it is finished downloading you will have a gzipped tar file.  This is a compressed file with lots of small files inside it.  To unpack use

`tar -zxvf grch38.tar.gz`

**Pull request merged**